pom.xml

```diff
@@ -17,17 +17,18 @@
     </scm>
 
     <properties>
-        <java.version>17</java.version>
+        <java.version>21</java.version>
         <maven.compiler.release>${java.version}</maven.compiler.release>
-        <spotbugs.version>4.7.3.3</spotbugs.version>
-        <jacoco.version>0.8.8</jacoco.version>
-        <owasp-check.version>8.2.</owasp-check.version>
+        <spotbugs.version>4.9.3.0</spotbugs.version>
+        <jacoco.version>0.8.13</jacoco.version>
+        <owasp-check.version>12.1.1</owasp-check.version>
 
-        <keycloak.version>20.0.5</keycloak.version>
+        <keycloak.version>26.2.0</keycloak.version>
```

Comment:
I think it is not up to date

Reply:
I can confirm that was not my priority as versions are often changing...